### ISSUE TITLE
Set session not working with node request module

### ISSUE THREAD

**Issue by 1nfinite** [expressjs/session#293]
I am using this module along with node request module for http request, in which the session value is stored in mongodb. I have found a weird scenario as follow. Not really sure whether there is something wrong in my code or is there a bug.

```
function changeSessionMaxAge (req, res) {
    req.session.cookie.maxAge = 36000; // new session maxAge value is set into mongodb

    request(option, function(err, response, body) {
        req.session.cookie.maxAge = 40000; // no error, but session maxAge in 
mongodb still remains 36000
        console.warn(req.session.cookie.maxAge) // display 40000
    });
}

```

**Comment by gabeio** [comment]
Is this still open or was this solved with #292?

**Comment by 1nfinite** [comment]
hi close. i just reaiise need to call save()
